README.md

```diff
@@ -83,24 +88,39 @@ We offer three installation methods:
    option, you will use pre-built container images that are available publicly,
    instead of building them yourself, which takes a long time).
 
-### Option 1: Running locally with “Docker for Desktop”
+### Option 1: Running locally
 
 > 💡 Recommended if you're planning to develop the application or giving it a
 > try on your local cluster.
 
 1. Install tools to run a Kubernetes cluster locally:
 
    - kubectl (can be installed via `gcloud components install kubectl`)
-   - Docker for Desktop (Mac/Windows): It provides Kubernetes support as [noted
+   - Local Kubernetes cluster deployment tool:
+        - [Minikube (recommended for
+         Linux)](https://kubernetes.io/docs/setup/minikube/).
+        - Docker for Desktop (recommended for Mac/Windows): It provides Kubernetes support as [noted
      here](https://docs.docker.com/docker-for-mac/kubernetes/).
    - [skaffold](https://skaffold.dev/docs/getting-started/#installing-skaffold)
      (ensure version ≥v0.20)
 
-1. Launch “Docker for Desktop”. Go to Preferences:
+1. Launch the local Kubernetes cluster with one of the following tools:
 
-   - choose “Enable Kubernetes”,
-   - set CPUs to at least 3, and Memory to at least 6.0 GiB
-   - on the "Disk" tab, set at least 32 GB disk space
+    - Launch Minikube (tested with Ubuntu Linux). Please, ensure that the
+       local Kubernetes cluster has at least:
+        - 4 CPU's
```

Comment:
more accurate would be `4 Hyperthreads`

Reply:
I don't think minikube uses this terminology.